server/rpm/pbench-server.spec

```diff
@@ -183,6 +189,14 @@ fi
 /%{installdir}/lib/pbench/server/utils.py
 /%{installdir}/lib/pbench/server/api/__init__.py
 /%{installdir}/lib/pbench/server/s3backup/__init__.py
+/%{installdir}/lib/pbench/cli/__init__.py
+/%{installdir}/lib/pbench/cli/agent/__init__.py
+/%{installdir}/lib/pbench/cli/agent/commands/__init__.py
+/%{installdir}/lib/pbench/cli/agent/commands/log/__init__.py
+/%{installdir}/lib/pbench/cli/agent/commands/results/__init__.py
+/%{installdir}/lib/pbench/cli/agent/options.py
```

Comment:
Do we need these "agent" files for the server?

Reply:
No - they are gone.